bindings/java/mongocrypt/README.md

```diff
@@ -0,0 +1,40 @@
+# mongocrypt Java Wrapper#
+The Java wrapper for the companion C library for client side encryption in drivers.
+
+
+### Testing ###
+`./gradlew clean check` runs the java test suite. By default it expects that libmongocrypt has been built and is in the `../../../cmake-build` directory.
+
+Note: libmongocrypt and the java library are [continuously built on evergreen](https://evergreen.mongodb.com/waterfall/libmongocrypt). Submit patch builds to this evergreen project when making changes to test on supported platforms.
+
+### Publishing ####
+
+**Snapshots**
+
+`./gradlew publishSnapshots` 
+Will push the latest snapshot version to the sonatype snapshot repository.
+
+**Releases**
+
+`./gradlew publishArchives` 
+Will push the latest snapshot version to maven central repository. 
```

Comment:
Not the latest snapshot version, but latest version? Current version?

Reply:
Done